### DISCUSSION TITLE
Image Status

### DISCUSSION THREAD

**Discussion by RS-JS** [chipzoller/hugo-clarity#309] [Q&A]
Hi, 

I know last time I updated my blog, back in March, there were quite a few things going on with images. I have had a look at issues and pull requests and can see a lot has been done in this area. I have a quite heavy image post that I am intending to put up this week and just wanted to check if it is 'safe to do so' or are there still ongoing issues?

**Comment by chipzoller** [comment]
Get the latest version and try your post. Let us know if anything is awry. If there is, you can just revert to your present version.
